README.md

```diff
@@ -1,122 +1,209 @@
-# Instana Go Collector
+# IBM Instana Go Tracer
 
-![Instana, an IBM company](https://user-images.githubusercontent.com/203793/135623131-0babc5b4-7599-4511-8bf0-ce05922de8a3.png)
+<!-- ![Instana, an IBM company](https://user-images.githubusercontent.com/203793/135623131-0babc5b4-7599-4511-8bf0-ce05922de8a3.png) -->
+[<img src="https://user-images.githubusercontent.com/203793/135623131-0babc5b4-7599-4511-8bf0-ce05922de8a3.png" width="410"/>](135623131-0babc5b4-7599-4511-8bf0-ce05922de8a3.png)
 
-[![Build Status](https://circleci.com/gh/instana/go-sensor/tree/master.svg?style=svg)](https://circleci.com/gh/instana/go-sensor/tree/main)
+[![Build Status](https://circleci.com/gh/instana/go-sensor/tree/main.svg?style=svg)](https://circleci.com/gh/instana/go-sensor/tree/main)
 [![PkgGoDev](https://pkg.go.dev/badge/github.com/instana/go-sensor)][pkg.go.dev]
 [![OpenTracing](https://img.shields.io/badge/OpenTracing-enabled-blue.svg)](http://opentracing.io)
 [![Go Report Card](https://goreportcard.com/badge/github.com/instana/go-sensor)](https://goreportcard.com/report/github.com/instana/go-sensor)
 
-The Go Collector is a runtime metrics collector, code execution tracer and profiler for applications and services written in Go. This module
-is a part of [Instana](https://instana.com) APM solution.
+The IBM Instana Go Tracer is an SDK that collects traces, metrics, logs and provides profiling for Go applications. The tracer is part of the [IBM Instana Observability](https://www.ibm.com/products/instana) tool set.
 
-## Dependency
-- Since version 1.47 the Go Collector requires Go version 1.13 or later.
-- Since version 1.53.0, the Go Collector uses fsm v1.0.1 internally. Customers using fsm version prior to v1 in their projects may face compilation issues and will need to update the fsm version to v1.
+## Compatibility
+
+|Tracer Version | Go version |
+|-----|-----|
+|Up to v1.46.0|v1.9.0 and higher|
+|v1.47.0 and higher|v1.13.0 and higher|
+
+> [!NOTE]
```

Comment:
These special directives work great on Github, but they are not rendered accordingly in pkg.go.dev.
We can remove it from the README.md if you see fit.
There is no need to remove them from other md files because only README.md is rendered in pkg.go.dev.

What are your thoughts?

Reply:
It looks pretty cool in Github. I wonder how it looks in pkg.go.dev though. If it is not too bad, I think we can keep it.

Reply:
I added it as a test to a personal repo. can you take a look? scroll down to the environment variables section: https://pkg.go.dev/github.com/willianpc/instana-mock-agent#section-readme @sanojsubran

Reply:
@willianpc It looks ok to me